src/app/(main)/academic-detail/components/AcademicDetailContent.tsx

```
@@ -1,94 +1,93 @@
-  
-          const result = await response.json();
-          setData({
-            semesterGrades: result.semesterGrades[0], // 해당 학기 성적
-            courses: result.courses, // 전공/교양 과목 목록
-          });
-        } catch (err) {
-          setError(err instanceof Error ? err.message : '알 수 없는 오류가 발생했습니다.');
-        } finally {
-          setIsLoading(false);
+  const searchParams = useSearchParams();
+  const year = parseInt(searchParams.get('year') || '0');
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)